index.js

```diff
@@ -0,0 +1,132 @@
+let interactedContainer = document.getElementById("interactedContainer")
+let interactionContainer = document.getElementById("interactionContainer")
+let userhandsimage = document.getElementById("userhandsImage")
+let computerhandsimage = document.getElementById("computerHandsImage")
+let gamestate = document.getElementById("gameState")
+let gamestatemobile = document.getElementById("gameStateMobile")
+let rules = document.getElementById("rulesPicture")
+let rulescontainer = document.getElementById("rulesContainer")
+let clickPaper = document.querySelector(".paper")
+let clickRock = document.querySelector(".rock")
+let clickScissors = document.querySelector(".Scissors")
+let rulesPop = document.getElementById("rules")
+let playagain = document.querySelectorAll(".playagain")
+clickPaper.addEventListener("click",paper)
+clickRock.addEventListener("click",rock)
+clickScissors.addEventListener("click",scissors)
+rulesPop.addEventListener("click",rulesPopup)
+playagain.forEach(button => {
+    button.addEventListener("click",playAgain)
+});
+let score = 0
+function randomNumber(){
+    let random = Math.floor(Math.random() * 3) + 1
+    if(random ===1){
+        computerhandsimage = document.getElementById("computerHandsImage").src = "assets/Rock.png"   
+    }
+    else if(random === 2){
+        computerhandsimage = document.getElementById("computerHandsImage").src = "assets/Paper.png"  
+    }
+    else if(random === 3){
+        computerhandsimage = document.getElementById("computerHandsImage").src = "assets/Scissors.png"  
+    }
+
+}
+function playAgain(){
+    interactionContainer.style.display = "flex"
+    interactedContainer.style.display = "none"
+    rulescontainer.style.display = "flex"
+}
+function paper(){
+    rulescontainer.style.display = "none"
+    let scoreboard = document.getElementById("scoreBoard")
+interactedContainer.style.display = "flex"
+interactionContainer.style.display = "none"
+userhandsimage = document.getElementById("userhandsImage").src = "assets/Paper.png"
+randomNumber()
+if(computerhandsimage === "assets/Paper.png"){
```

Comment:
Looks like `computerhandsimage` isn't consistent. On L4 it refers to a DOM element and from the `randomNumber` function it's now the image name. 

While you can have it work this way, as the project grows these inconsistences often lead to bugs.

How about have the `randomNumber()` function return the image string and then you can use it here? Instead of relying on the global state change.

```js 
let computerImage = pickRandomImage()
``` 

_Note: I renamed `randomNumber` to `pickRandomImage` here_

Reply:
Very true
I must have made you read the entire code to figure out what was going on.